app/views/dashboard/show.html.erb

```diff
@@ -1,11 +1,29 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <h1 class="govuk-heading-l">User dashboard</h1>
-    <p class="govuk-body"> You are logged in</p>
+
 
     <% if @current_user.lead_provider? %>
-      <%= govuk_link_to "Go to school search", school_search_path %>
-    <% end %>
 
+      <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+          <%= @lead_provider.name %>
+        </h1>
+        <div class="index-group index-group-lead-provider">
+          <h2 class="govuk-heading-m">
+            <%= govuk_link_to "Find and add schools", lead_providers_your_schools_path %>
```

Comment:
I don't think this link goes here?

Reply:
No - this is chicken and the egg scenario - because the page links to the code on your side I'll fix it after your code gets merged - I'll just create a small task to update the links

Reply:
Added TODO and set the path to nil